src/main/java/Store.java

```diff
@@ -0,0 +1,263 @@
+                        Timing = Date + " " + Time;
+                    } catch (DateTimeException d) {
+                        DE.InvalidTimeFormat();
+                        return;
+                    }
+                } else {
+                    Timing = Date;
+                }
+                Task T = new Event(cmd, Timing);
+                Storage.add(T);
+                T.Output();
+                System.out.println(String.format("Now you have %d tasks in the list.", counter));
+                System.out.print(ui.line());
+                WritetoFile();
+            } catch (DateTimeException d) {
+                DE.InvalidDateFormat();
+            }
+        }
+    }
+
+    /**
+     * This method remove the Task at the index from the ArrayList.
+     * @param index Indicates which item on the ArrayList<Task> Storage.
+     */
+    public void delete(int index){
+        if(index > Storage.size() || index <= 0){
+            DE.ExceedList();
+        } else {
+            Task UpdateCurrAction = Storage.get(index - 1);
+            String CurrAction = UpdateCurrAction.toString();
+            int amt = Storage.size() - 1;
+            Storage.remove(index - 1);
+            System.out.print(ui.DeleteMessage(CurrAction, amt));
+        }
+        WritetoFile();
+    }
+
+    /**
+     * This method writes the ArrayList<Task> to the file on the hard disk.
+     */
+    public void WritetoFile() {
+        try {
+            FileWriter fileWriter = new FileWriter(file);
+            for (int i = 0; i < Storage.size(); i++) {
+                String data = String.format("%d.", i + 1) + Storage.get(i).toString();
+                fileWriter.write(data);
+                fileWriter.write("\n");
+            }
+            fileWriter.close();
+        } catch (IOException e){
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * This method load existing file onto the ArrayList<Task> Storage.
+     * @param S This is the Task from the file to be loaded from.
+     */
+    public void load(String S){
+        boolean R = CheckIfDone(S);
+        if(S.contains("[T]")){
+            String result = S.substring(9);
+            Task T = new Todo(result);
+            T.isDone = R;
+            Storage.add(T);
+            WritetoFile();
+        } else if (S.contains("[D]")){
+            String NewInput = S.substring(9);
+            String[] AT = NewInput.split("\\|");
+            String time = AT[1].strip().substring(3).strip();
+            Task T = new Deadline(AT[0], time);
+            T.isDone = R;
+            Storage.add(T);
+            WritetoFile();
+        } else if (S.contains("[E]")){
+            String result = S.substring(9);
+            String[] AT = result.split("\\|");
+            String time = AT[1].strip().substring(3).strip();
+            Task T = new Event(AT[0], time);
+            T.isDone = R;
+            Storage.add(T);
+            WritetoFile();
+        }
+    }
```

Comment:
Overall, nice implementation of the code! Good separation of methods make the code easy to follow! :-) One suggestion I have is to try to follow the JavaDoc standard for variable and method naming (camelCase e.g. boolean r, writeToFile())! But other than that good implementation of the code!